streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/worker/context/StreamingRuntimeContext.java

```diff
@@ -0,0 +1,98 @@
+package io.ray.streaming.runtime.worker.context;
+
+import io.ray.streaming.api.context.RuntimeContext;
+import io.ray.streaming.runtime.core.graph.executiongraph.ExecutionVertex;
+import io.ray.streaming.state.backend.KeyStateBackend;
+import io.ray.streaming.state.keystate.desc.ListStateDescriptor;
+import io.ray.streaming.state.keystate.desc.MapStateDescriptor;
+import io.ray.streaming.state.keystate.desc.ValueStateDescriptor;
+import io.ray.streaming.state.keystate.state.ListState;
+import io.ray.streaming.state.keystate.state.MapState;
+import io.ray.streaming.state.keystate.state.ValueState;
+import java.util.Map;
+
+/**
+ * Use Ray to implement RuntimeContext.
+ */
+public class StreamingRuntimeContext implements RuntimeContext {
```

Comment:
This class is duplicated with `RayRuntimeContext ` ?

Reply:
Remove ```RayRuntimeContext```.